sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/HttpInstrumentationPolicy.java

```diff
@@ -169,6 +171,8 @@ public final class HttpInstrumentationPolicy implements HttpPipelinePolicy {
     private static final String REQUEST_DURATION_METRIC_NAME = "http.client.request.duration";
     private static final String REQUEST_DURATION_METRIC_DESCRIPTION = "Duration of HTTP client requests";
     private static final String REQUEST_DURATION_METRIC_UNIT = "s";
+    private static final List<Double> REQUEST_DURATION_BOUNDARIES_ADVICE
```

Comment:
From an offline discussion, this is something specified by OpenTelemetry, please link to their definition so this is more easily understood.

Reply:
added comments